src/fsharp/FSharp.Core/string.fs

```diff
@@ -31,12 +31,15 @@ namespace Microsoft.FSharp.Core
 
         [<CompiledName("Map")>]
         let map (mapping: char -> char) (str:string) =
-            if String.IsNullOrEmpty str then
-                String.Empty
+            if String.IsNullOrEmpty str then String.Empty
```

Comment:
Can we please keep the string.empty with same indentation as before?

Reply:
Sure, will fix

Reply:
Done: https://github.com/dotnet/fsharp/pull/9470/commits/6c6b6721c522bb0ceca0ea22142396178ed6e7f3